packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -832,6 +832,7 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
 			}
 		}
 
+		//* Sidenote: Does this result in duplicates if the input overlaps with the existing Tombstone list?
```

Comment:
It probably does yes. Maybe `this.tombstonedBlobs` should be a set.

Reply:
Oh it is a `Set` -_-

Reply:
But more importantly... `this.tombstoneBlobs` is never read from!  Probably from the recent change to centralize Tombstone detection to GC.  I'll open a small PR to clean this up, in case it's that simple.

Reply:
Actually leaving it, there's not really any harm, I updated the description of AB#6064 `GC: Consolidate Tombstone/Delete logging/throwing logic in GC classes` to mention this.